sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -70,6 +70,17 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   import hiveContext._
   import spark.implicits._
 
+  private var threadContextClassLoader: ClassLoader = _
+
+  override protected def beforeEach(): Unit = {
```

Comment:
This is needed to make a new UT pass, as some tests change the current thread's context classloader to jar classloader.